<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Entregar productos</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>641</width>
     <height>361</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Productos</string>
    </attribute>
    <widget class="QListWidget" name="list">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>641</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_mostrar">
     <property name="geometry">
      <rect>
       <x>464</x>
       <y>240</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Mostrar Productos </string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_entregar">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>280</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Entregar productos</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Productos entregados</string>
    </attribute>
    <widget class="QListWidget" name="list2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>641</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_mostrar2">
     <property name="geometry">
      <rect>
       <x>504</x>
       <y>270</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Mostar datos</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
